helm_deploy/values-dev.yaml

```
@@ -5,7 +5,7 @@ generic-service:
   replicaCount: 2
 
   ingress:
-    host: suicide-risk-form-ui-dev.hmpps.service.justice.gov.uk
+    host: suicide-risk-form-dev.hmpps.service.justice.gov.uk
 
   env:
     INGRESS_URL: "https://suicide-risk-form-ui-dev.hmpps.service.justice.gov.uk"
```

This should match the ingress host above
```suggestion
    INGRESS_URL: "https://suicide-risk-form-dev.hmpps.service.justice.gov.uk"
```